06_modules_lab/01.py

```diff
@@ -1,4 +1,6 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+import sys
+
+numOfPrints = int(sys.argv[1])
+
+for x in range(0,numOfPrints):

```

Comment:
You're not using the `x` nor the start value (0), so better to omit both. This works just as well:

```
for _ in range(numOfPrints):
```